src/wasm-lib/kcl/src/lib.rs

```diff
@@ -60,7 +60,7 @@ mod coredump;
 mod docs;
 mod engine;
 mod errors;
-mod execution;
+pub mod execution;
```

Comment:
nrc went through and made everything less public to make the API surface area smaller and make changes easier. Instead of making the entire module public, either add it to the `pub use execution::{` statement or `pub mod exec {` below. Same for `std`. There's just too much in there.

```suggestion
mod execution;
```